src/main/java/com/aws/greengrass/mqtt/bridge/MessageBridge.java

```diff
@@ -111,7 +111,7 @@ private void handleMessage(TopicMapping.TopicType sourceType, Message message) {
                     Message msg = new Message(targetTopic, message.getPayload());
                     try {
                         client.publish(msg);
-                        LOGGER.atInfo().kv(LOG_KEY_SOURCE_TYPE, sourceType).kv(LOG_KEY_SOURCE_TOPIC, fullSourceTopic)
+                        LOGGER.atTrace().kv(LOG_KEY_SOURCE_TYPE, sourceType).kv(LOG_KEY_SOURCE_TOPIC, fullSourceTopic)
```

Comment:
Trace is too low. Debug is fine for this

Reply:
I wouldn't really agree here. Trace should be for tracing individual messages (which is what this log is). Debug can be more commonly enabled and it should provide more details, but not necessarily more frequency.

Reply:
We need visibility into this when customers aren't seeing their messages flowing how they expect. Trace isn't even a documented log level.

Needs to be debug